projects/plugins/jetpack/extensions/blocks/cookie-consent/attributes.js

```diff
@@ -0,0 +1,43 @@
+import { __ } from '@wordpress/i18n';
+
+export default {
+	text: {
+		type: 'string',
+		source: 'html',
+		selector: 'p',
+		default: __(
+			'Privacy & Cookies: This site uses cookies. By continuing to use this website, you agree to their use. To find out more, including how to control cookies, see here: <a href=https://automattic.com/cookies/>Cookie Policy</a>.',
+			'jetpack'
+		),
+	},
+	style: {
+		type: 'object',
+		default: {
+			color: {
+				text: 'var(--wp--preset--color--contrast)',
+				background: 'var(--wp--preset--color--tertiary)',
+				link: 'var(--wp--preset--color--contrast)',
```

Comment:
Something seems off with this. It grabs the colors just fine, but when I go to edit them, I see this:

<img width="583" alt="image" src="https://user-images.githubusercontent.com/426388/221943871-795d4b97-7c15-4ef6-84dd-7ce61085b37f.png">

Reply:
I couldn't come up with a way to make this work. Undocumented and I tried everything I can come up with!